client/my-sites/patterns/components/grid-gallery/style.scss

```
@@ -0,0 +1,49 @@
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.patterns-grid-gallery {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 0 24px;
+	margin-bottom: -32px;
+
+	.patterns-grid-gallery__item {
+		flex: 1 0 var(--item-min-width);
+		padding-bottom: 32px;
```

```suggestion
	gap: 32px 24px;

	.patterns-grid-gallery__item {
```
With grid layout, we can also do this to simplify the spacing logic.

---

You are right 👍